google/auth/external_account_authorized_user.py

```diff
@@ -333,15 +333,15 @@ def revoke(self, request):
             google.auth.exceptions.OAuthError: If the token could not be
                 revoked.
         """
-        if not self._revoke_url or not self._refresh_token_val:
+        if not self._revoke_url or not self._refresh_token:
             raise exceptions.OAuthError(
                 "The credentials do not contain the necessary fields to "
                 "revoke the refresh token. You must specify revoke_url and "
                 "refresh_token."
             )
 
         self._sts_client.revoke_token(
-            request, self._refresh_token_val, "refresh_token", self._revoke_url
+            request, self._refresh_token, "refresh_token", self._revoke_url
         )
         self.token = None
         self._refresh_token = None
```

Comment:
This appears to be a bug in the previous code. I think this was supposed to be `self._refresh_token_val`.

This PR should fix any issue here though